### PR TITLE
Bug 1130410 - Remove optimize-clean.js from post-app.js

### PR DESCRIPTION
**Pull request by rickychien**
https://bugzilla.mozilla.org/show_bug.cgi?id=1130410

### CODE REVIEW

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a329ce06965a)

File: build/optimize-clean.js [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-      webapp.sourceDirectoryName != options.BUILD_APP_NAME) {
-    return;
-  }
-
-  let re = new RegExp('\\.html\\.' + options.GAIA_DEFAULT_LOCALE + '$');
-  let files = utils.ls(webapp.buildDirectoryFile, true);
-  files.forEach(function(file) {
-    if (
-      re.test(file.leafName) ||
-      file.leafName.indexOf(utils.gaia.aggregatePrefix) === 0
```
Comment (timdream) [inline build/optimize-clean.js]: This is the last reference of `utils.gaia.aggregatePrefix` outside of `webapp-optimize.js`. Do we want to keep it?

Comment (RickyChien) [inline build/optimize-clean.js]: I think we should keep it in order to filter out gaia_build_xxx files in build_stage.

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e9c251977b94)

**Pull request merged**